src/main/java/laddergame/domain/connectiongenerator/RandomConnectionGenerator.java

```diff
@@ -0,0 +1,41 @@
+package laddergame.domain.connectiongenerator;
+
+import laddergame.domain.ladder.Connection;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static laddergame.domain.ladder.Connection.CONNECTED;
+import static laddergame.domain.ladder.Connection.NOTCONNECTED;
+
+public class RandomConnectionGenerator implements ConnectionGenerator {
+    private static final double PERCENTAGE_OF_CONNECTION = 0.5;
+
+    @Override
+    public List<Connection> getConnections(int peopleNumber) {
+        int connectionNumber = peopleNumber - 1;
+        Connection beforeConnection = addRandomConnection();
+
+        return Stream.iterate(beforeConnection, this::addConnection)
+                .limit(connectionNumber)
+                .toList();
+    }
+
+    private Connection addConnection(Connection beforeConnection) {
+        if (beforeConnection==CONNECTED) {
+            beforeConnection = NOTCONNECTED;
+            return beforeConnection;
+        }
+
+        beforeConnection = addRandomConnection();
+        return beforeConnection;
+    }
+
+    private Connection addRandomConnection() {
+
+        if(Math.random() <= PERCENTAGE_OF_CONNECTION){
```

Comment:
```suggestion
        if(Math.random() <= PERCENTAGE_OF_CONNECTION) {
```
또한 `util.Random`에 `nextBoolean`도 있습니다~!

Reply:
random seed값에 따라 완전한 random값이 아니라는 크루들의 의견이 있었는데 썬의 피드백으로 이 고민이 해결된 것 같아요!! 적용해주었습니다